### DISCUSSION TITLE
ERR 500 无法加载网盘内容

### DISCUSSION THREAD

**Discussion by shuoyuan** [spencerwooo/onedrive-vercel-index#146] [Q&A] [answered]
正常使用一段时间后，报 ERR 500
![image](https://user-images.githubusercontent.com/25116465/137100208-04ec0bfa-50b5-4515-9114-0fbb406ba6c6.png)

网页登陆Onedrive可以正常访问文件夹和文件。

**Comment by shuoyuan** [comment, marked as answer]
看了一下 Vercel 的 Realtime Logs，发现是我的 Token 过期了 --！
重新获取 Refresh Token 和 Access Token 之后成功部署。

--

**Comment by myl7** [comment]
Ref: issue #187
